resources/lang/de/authorization.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'beatmap_discussion' => [
+        'destroy' => [
+            'is_hype' => 'Hypen kann nicht rückgängig gemacht werden.',
+            'has_reply' => 'Diskussion mit Antworten kann nicht gelöscht werden',
+        ],
+        'nominate' => [
+            'exhausted' => 'Dein Nominierungslimit für heute wurde erreicht, bitte versuche es morgen erneut.',
+        ],
+        'resolve' => [
+            // todo: gute übersetzung für 'resolve'..
+            // diskussion für beendet / gelöst erklären? auflösen? klären?
+            'not_owner' => 'Nur der Threadersteller und Beatmapbesitzer können die Diskussion .',
+        ],
+
+        'vote' => [
+            'limit_exceeded' => 'Bitte warte eine Weile, bevor du mehr Stimmen abgibst',
+            'owner' => 'Man kann nicht bei der eigenen Diskussion abstimmen!',
```

Comment:
`Man kann nicht in der eigenen Diskussion abstimmen`?

Reply:
Oh. Ja, auf jeden Fall.